lang/python/python3-sqlparse/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
```

Comment:
Remove it.  For this package, there is https://github.com/openwrt/packages/pull/10375.